README.md

```diff
@@ -83,5 +87,18 @@ array(2) {
 }
 ```
 
+In order to validate URL, use the RobotsTxtValidator class:
+```php
+$parser = new RobotsTxtParser(file_get_contents('http://example.com/robots.txt'));
+$validator = new RobotsTxtValidator($parser->getRules());
+
+$url = '/';
+$userAgent = 'MyAwesomeBot';
+if($validator->isUrlAllow($url, $userAgent)){
```

Comment:
`if ($validator->isUrlAllow($url, $userAgent)) {`
Just for beauty and code style, could you please add additional whitespaces?